device_path: "/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0" # Path to receiver port
baud_rate: 115200 # baud rate
frame_id: "imu_link"
